src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceDeserializer.cs

```diff
@@ -384,6 +384,24 @@ public virtual object CreateResourceInstance(IEdmStructuredTypeReference structu
             }
         }
 
+        /// <summary>
+        /// Deserializes the instance annotations from <paramref name="resourceWrapper"/> into <paramref name="resource"/>.
+        /// </summary>
+        /// <param name="resource">The object into which the structural properties should be read.</param>
```

Comment:
>structural properties [](start = 61, length = 21)

"annotations"

Reply:
#Resolved

Reply:
@  - looks still not fixed (comment references structural properties, not annotations, here and on line 393).

---
In reply to: [458438030](https://github.com/OData/WebApi/pull/2219#discussion_r458438030) [](ancestors = 458438030)